src/utils.js

```diff
@@ -58,6 +58,57 @@ export function getRootExpression(node) {
 }
 
 /*
+ * Returns the name of the reference node passed.
+ *
+ * example: var foo = document;
+ *  The node for foo will return 'document'
```

Comment:
Oh, cool. Could you add that to the doc? The extra line adds a lot of context :smile: